docs/.env.example

```diff
@@ -1 +1,20 @@
-NEXT_PUBLIC_APP_URL=http://localhost:3004
+NEXT_PUBLIC_APP_URL=
```

Comment:
We should only have the essentials:

DOCS_URL=http://localhost:3000 
APP_URL=http://localhost:3001